src/bentoml/_internal/configuration/helpers.py

```diff
@@ -105,17 +105,19 @@ def get_default_config(version: int) -> dict[str, t.Any]:
         )
     )
     mod = import_configuration_spec(version)
-    assert hasattr(mod, "SCHEMA"), (
-        "version %d does not have a validation schema" % version
-    )
     try:
         mod.SCHEMA.validate(config)
     except s.SchemaError as e:
         raise BentoMLConfigException(
             "Default configuration for version %d does not conform to given schema:\n%s"
             % (version, e)
-        ) from None
-    return config
+        ) from e
+    except Exception as err:
+        logger.error("Unexpected error while setting configuration:\n")
+        logger.error(err)
```

Comment:
Why are we both printing and raising here?

Reply:
hmm, i think that I want to show the exception here instead of printing.